modules/bullet/collision_object_bullet.h

```diff
@@ -183,13 +190,21 @@ class CollisionObjectBullet : public RIDBullet {
 		return collisionLayer & p_other->collisionMask || p_other->collisionLayer & collisionMask;
 	}
 
-	virtual void reload_body() = 0;
+	bool need_reload_body() const {
+		return need_body_reload;
+	}
+
```

Comment:
Again, `need_reload_body()` doesn't appear to be used.

Reply:
Yes, but it's the getter of a variable, it's there for code completeness.